lib/FrontendTool/FrontendTool.cpp

```diff
@@ -2038,6 +2038,11 @@ int swift::performFrontend(ArrayRef<const char *> Args,
     return finishDiagProcessing(1, /*verifierEnabled*/ false);
   }
 
+  Optional<llvm::PrettyStackTraceString> allowErrorsStackTrace;
+  if (Invocation.getFrontendOptions().AllowModuleWithCompilerErrors)
+    allowErrorsStackTrace.emplace("While allowing modules with compiler errors "
+                                  "enabled");
```

Comment:
Is this check for any frontend jobs and we'd expect the prettystacktrace only for the background indexing then right?

Reply:
Yep, we should only see this message in background indexing processes.